### DISCUSSION TITLE
XSD Validation Failed in notice SubType 16

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#402] [Q&A] [answered]
Hi all !
Did anyone try notice subtype 16 ?
I am using sdk 1.6, and I get 2 xsd validation error  : 

[SC=BA232AUSR, IC=01000, VID=33944abc-a779-4b9d-b132-1b1b8b367495, AID=21862, UID=46104] - ERROR:: at Line:     1
Column:         4037
Message:        cvc-complex-type.2.4.a: Invalid content was found starting with element '{"urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2":DocumentType}'. One of '{"urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2":UBLExtensions, "urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2":ID}' is expected.

2023-04-04 10:08:33,541 ERROR - 18.156.166.235 - /sdk/notice/en/sendToValidation.si - [ja-dev-as06.de.jaggaer.com#default task-10#00014] (com.bs.esop.labs.noticeforms.mains.SdkTools lambda$5 563)
[SC=BA232AUSR, IC=01000, VID=33944abc-a779-4b9d-b132-1b1b8b367495, AID=21862, UID=46104] - ERROR:: at Line:     1
Column:         4037
Message:        cvc-complex-type.3.2.2: Attribute 'listName' is not allowed to appear in element 'cbc:DocumentType'.

The first is 'easy' .. it lacks the ID (ID – Procurement documents), but the second , seems due to definition  of BT-14-Lot , unless for any reason one does not set the attribute listName.
Thanks a lot
Renato
![Notice16](https://user-images.githubusercontent.com/52211037/229765712-281b5a54-161f-4a0f-a8de-05c1285b19cf.jpg)

**Comment by YvesJo** [comment, marked as answer]
Hi,
Your analysis is correct. The XML element "cbc:DocumentType" associated to BT-14 is of type Text and does therefore not have any listName attribute; this attribute shouldn't therefore be specified. This is a similar situation to the one of BT-01.
